examples/react/navigation-blocking/package.json

```diff
@@ -9,10 +9,8 @@
     "start": "vite"
   },
   "dependencies": {
-    "@tanstack/react-query": "^5.17.8",
```

Comment:
I'd keep these packages in.

They were kept in, since in the future we may want to expand these examples to asynchronous-based actions.